gallery/src/PageHeader/PageHeader.tsx

```diff
@@ -29,7 +30,9 @@ const PageHeader = () => (
       <span className="gallery-page-header__name">
         React Shared Component Library
       </span>
-
+      <div className="gallery-page-header__display-mode-btn">
```

Comment:
Is the extra div needed?

Reply:
> Is the extra div needed?

Nope, it works fine without. Removed here: [da74491](https://github.com/sonatype/sonatype-react-shared-components/pull/782/commits/da7449165643e62842d939367922afd0b4c11f25)